services/order.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        tickets_ = []
+        user = get_user_model().objects.get(username=username)
+        order = Order(user=user)
+        if date:
+            date_transform = datetime.strptime(date, "%Y-%m-%d %H:%M")

```

Comment:
Ensure that the date format in `datetime.strptime(date, "%Y-%m-%d %H:%M")` matches the expected input format. Consider using timezone-aware datetime objects if your application requires it.